helm-charts/splunk-otel-collector/templates/_helpers.tpl

```diff
@@ -221,38 +221,41 @@ Create the patch-log-dirs image name.
 {{- end -}}
 
 {{/*
-Convert memory value from resources.limit to numeric value in MiB to be used by otel memory_limiter processor.
-*/}}
-{{- define "splunk-otel-collector.convertMemToMib" -}}
+  This helper converts the input value of memory to MiB.
+  Input needs to be a valid value as supported by k8s memory resource field.
+ */}}
+{{- define "splunk-otel-collector.convertMemToMib" }}
```

Comment:
The template converts the input to bytes first and then bytes to MiB. Check the unttest file for examples.